frontend/src/components/icons/HomeIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
```

Comment:
Needs deleting now that it's done like this `import {ReactComponent as HomeIcon} from '../assets/icons/home.svg'`